.oca-port.json

```diff
@@ -0,0 +1,11 @@
+{
+  "14.0": {
+    "16.0-mig-l10n_fr_hr_check_ssnid": {
+      "l10n_fr_hr_check_ssnid": {
+        "blacklist_pull_requests": {
+          "302": true
+        }
+      }
+    }
+  }
+}
```

Comment:
Using an old version of oca-port, maybe?
AFAIK this file should be in `.oca/` subfolder, with the latest version

Reply:
Probably @syera94 could you update oca-port ? Thanks

Reply:
no more old version data